common/config/rush/common-versions.json

```diff
@@ -12,7 +12,6 @@
    * SemVer range specifier.  See the Rush documentation for details about this feature.
    */
   "preferredVersions": {
-
```

Comment:
We should just unstage all changes to this file since we're not actually modifying it.

Reply:
Done!